R/pivot-long.R

```diff
@@ -282,16 +287,24 @@ pivot_longer_spec <- function(data,
     names[col_id] <- cols
 
     names(val_cols) <- names
-    val_type <- vec_ptype_common(!!!val_cols[col_id], .ptype = values_ptypes[[value]])
-    val_cols <- vec_cast_common(!!!val_cols, .to = val_type)
+    val_type <- vec_ptype_common(
+      !!!val_cols[col_id],
+      .ptype = values_ptypes[[value]],
+      .call = error_call
+    )
+    val_cols <- vec_cast_common(
+      !!!val_cols,
+      .to = val_type,
+      .call = error_call
+    )
     val_cols <- unname(val_cols)
 
     if (cols_vary == "slowest") {
       vals[[value]] <- list_unchop(val_cols, ptype = val_type)
     } else if (cols_vary == "fastest") {
       vals[[value]] <- vec_interleave(!!!val_cols, .ptype = val_type)
     } else {
-      cli::cli_abort("Unknown {arg cols_vary} value.", .internal = TRUE)
+      cli::cli_abort("Unknown {.arg cols_vary} value.", .internal = TRUE)
```

Comment:
Oops

Reply:
this one was even better https://github.com/tidyverse/tidyr/pull/1436/commits/d66d62aaa1e1b472549315b55b5bf5b19eb8818a